cmd/yggdrasil/main.go

```diff
@@ -312,6 +312,22 @@ func main() {
 		}
 	}
 
+	// Promise final modes of operation.  At this point, if at all:
+	// - raw socket is created/open
+	// - admin socket is created/open
+	// - privileges are dropped to non-root user
+	promises := []string{"stdio", "inet", "dns"}
+	if strings.HasPrefix(cfg.AdminListen, "unix://") {
```

Comment:
What about `Peers`, `InterfacePeers` or `Listen`? Those can be `unix://` too.

Reply:
Right, I'll fix that and update the PR, thanks.